assert/assertions.go

```diff
@@ -900,6 +907,8 @@ func containsElement(list interface{}, element interface{}) (ok, found bool) {
 //	assert.Contains(t, "Hello World", "World")
 //	assert.Contains(t, ["Hello", "World"], "World")
 //	assert.Contains(t, {"Hello": "World"}, "Hello")
+//	assert.Contains(t, []byte("Hello World"), []byte("World"))
+//	assert.Contains(t, [[]byte("Hello"), []byte("World")], []byte("World"))
```

Comment:
This line is not valid Go.

Also: any case presented in the doc must be strictly checked in the test suite.